src/dynapi/SDL_dynapi_overrides.h

```diff
@@ -625,6 +625,7 @@
 #define SDL_MemoryBarrierAcquireFunction SDL_MemoryBarrierAcquireFunction_REAL
 #define SDL_JoystickGetDeviceInstanceID SDL_JoystickGetDeviceInstanceID_REAL
 #define SDL_utf8strlen SDL_utf8strlen_REAL
+#define SDL_utf8strllen SDL_utf8strllen_REAL
```

Comment:
Run gendynapi.pl in the same folder as this file and commit the result, should produce a working equivalent of this.